tools/docker-compose/Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM centos:7
+FROM fedora:27
```

Comment:
What's the reasoning behind this change?  Some newer dependency that's not in CentOS?

Reply:
Ugh, yeah.  Django 2.2 bumped the dependency on sqlite3 to 3.8.3, when you are using sqlite as the backend.  CentOS 7 is still stuck back at 3.7.17.

Reply:
Is there a reason we don't just host a CentOS7 sqlite update?

Reply:
We put a bit of effort into trying to just use a version direct off of the sqlite website, but the problem is that Python itself has to be made aware of the new version which seems like it may require a recompile.

Reply:
Hm. Library soname is the same.

This is going to be a problem for EL7 builds....

Reply:
@wenottingham This is the image for the dev container. What kind of problem do you foresee happening here? The tests are passing, so what am I misunderstanding? Updating sqlite alone was unfortunately not enough since it's using some sqlite python binding that was included in the base os (it seems). We can switch to CentOS 8 when that comes out. I'm also open to other solutions if there is one we might have missed.

Reply:
Oh, dev container only. Don't mind me.

Reply:
Well, aside from F27 being EOL. Probably should be 29 or 30.

Reply:
There was a reason I had to drop it back down from 30.  Some other system dependency that we rely on was no longer available.

Reply:
I'm fine investigating 29/30, but let's do that in another PR after this is in (given that this is dev-env only).

Reply:
Could we drop a comment in this file capturing some of this background info?